OpenRA.Mods.Common/Widgets/Logic/Editor/ActorSelectorLogic.cs

```diff
@@ -137,7 +137,7 @@ void IntializeActorPreviews()
 					item.IsVisible = () => true;
 
 					var tooltip = actor.TraitInfoOrDefault<TooltipInfo>();
-					item.GetTooltipText = () => (tooltip == null ? "\nType: " : tooltip.Name + "\nType: ") + actor.Name;
+					item.GetTooltipText = () => (tooltip == null ? " < " + actor.Name + " > " : tooltip.Name) + "\nType: " + actor.Name;
```

Comment:
`actor.Name` is included in the next line, so I think it would be better to just omit this first line completely if the tooltip is null.